user-guide/query-examples-hacks/redash-hacks.md

```diff
@@ -81,5 +82,13 @@ CASE
         WHEN '{{date}}' = 'All' THEN date IS NOT NULL
         ELSE date = '{{date}}'
   END
-  ```
+```
 Kelly (thanks Kelly!) from Hudl shared this in our [Slack community](http://slack.redash.io).
+
+## Adding Tags to Queries {#query-tags}
+
+Somewhere in the middle between naming queries with long-tediou-very descriptive names and having fancy clickable tags we neutrally present to you - #tags!
```

Comment:
"present to you" feels like something you would write in a blog post that introduces a feature rather than timeless help article.